src/msg/msg_utils.rs

```diff
@@ -50,6 +50,30 @@ impl From<String> for TokenIdPair {
     }
 }
 
+// TODO:
+fn hash_order(_order: &crate::types::matchengine::messages::Order) -> String {
```

Comment:
why return String here rather than BigInt/Fr?

Reply:
fixed in https://github.com/Fluidex/rollup-state-manager/pull/80/commits/727648de2cf9c372477f870fa95690acda3848e6